.github/workflows/code_review.yml

```diff
@@ -0,0 +1,22 @@
+name: AI Code Reviewer
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+permissions: write-all
+jobs:
+  review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: AI Code Reviewer
+        uses: lfsevergnini/ai-codereviewer@chore/add-model-version-parameter
+        with:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_TOKEN }}
+          OPENAI_API_MODEL: gpt-3.5-turbo
```

Comment:
The `OPENAI_API_MODEL` parameter has been added to the `AI Code Reviewer` step. Please provide a justification for using the `gpt-3.5-turbo` model instead of the default model.

Reply:
the default is `gpt-4`, but that is not available for my account